### DISCUSSION TITLE
Help user search library by topic

### DISCUSSION THREAD

**Discussion by IronsDu** [microsoft/vcpkg#23083] [Q&A]
Can I add lib of some topic?
It can be helpfully other user search my lib by some topic, not only by lib's name.

**Comment by sdcb** [comment]
Useful, I think we can implement it by add a tag row in CONTROL file such as:

> Source: cpprestsdk
> Version: 2.10.0
> **Tags: network; http**

**Comment by ras0219-msft** [comment]
Currently, we perform a substring search into the package description as well, so i think the best first step in this direction would be to add a list of tags to the descriptions of packages.
